### PR TITLE
Gazebo urdf improvements

### PR DESCRIPTION
**Pull request by jrgnicho**
This PR uses a xacro to create urdf files for gazebo and for previewing motion
It also creates a soft link into the .gazebo/models directory so that gazebo can find the robot mesh resources

### CODE REVIEW

**Comment by mpowelson** [comment]
While I understand why you added it, I don't think rosdep will actually pull libompl-dev since there is no key for it. Personally I am fine with leaving it in the package.xml to make people aware of what they need to install, but I don't know if that is best practice. If you do, you'll need to add it to the travis.yaml ROSDEP_SKIP_KEYS in order for CI to pass.

File: crs_support/launch/environment_launch.launch.py [hunk 1]
```diff
@@ -7,25 +8,49 @@
...
+    cmd1 = 'ln -s %s %s'%(crs_model_path, gazebo_model_path)
+    if not os.path.exists(gazebo_model_path):
+        cmd_dict[cmd1] = True        
+    
+    required_cmd = False
     try:    
-        cmd = 'killall -9 gazebo & killall -9 gzserver & killall -9 gzclient'
-        process = subprocess.run(cmd , shell=True, check=True, stdout=subprocess.PIPE, universal_newlines=True)
+
+        for cmd, req_ in cmd_dict.items():
```
Comment (marrts) [inline crs_support/launch/environment_launch.launch.py]: The try-except should be inside the for loop. The way it is currently set up is if the cmd4 fails, which it often does if gazebo isn't running, then it won't even try cmd1.

**Review by marrts — CHANGES_REQUESTED** [review]
Looks good, much better way of making sure the gazebo models directory is populated and the preview joint_state now works smoothly with the regular joint_state. Only change is that the link between crs_support and .gazebo/models doesn't always work as currently implemented.

**Comment by jrgnicho** [comment]
@mpowelson  is the ROSDEP_SKIP_KEYS a new file that I'd have to create in the repo?

**Comment by jrgnicho** [comment]
> Looks good, much better way of making sure the gazebo models directory is populated and the preview joint_state now works smoothly with the regular joint_state. Only change is that the link between crs_support and .gazebo/models doesn't always work as currently implemented.

I haven't had the `ln -s ...` command fail on me yet.  Does it not create the link?

**Pull request merged**